handlers/handler-tracing-zipkin/src/main/java/org/apache/servicecomb/tracing/zipkin/ZipkinConsumerDelegate.java

```diff
@@ -50,7 +50,7 @@ class ZipkinConsumerDelegate implements ZipkinTracingDelegate {
   }
 
   @Override
-  public Span createSpan(Invocation invocation) {
+  public synchronized Span createSpan(Invocation invocation) {
     return handler.handleSend(requestWrapper.invocation(invocation));
   }
```

Comment:
This fix seems not correct. HttpClientResponseWrapper/HttpClientRequestWrapper should be invocation scope, not bean scope.  Synchronization can not fix this problem .

Reply:
Based on my understanding of the brave's source code, I think only this part can set context to invocation.
![image](https://github.com/apache/servicecomb-java-chassis/assets/73453250/e3af5482-a68e-4826-8680-673d2430b235)
When the handler. handleSend() method is called, it will insert the context related to trace into the invocation.
![image](https://github.com/apache/servicecomb-java-chassis/assets/73453250/1f586525-1813-442f-a3ae-3c1d396c1614)
![image](https://github.com/apache/servicecomb-java-chassis/assets/73453250/204b068c-a9a8-4e61-8f33-0533b6aa7fae)

And after I added synchronized and tested again, there was no further loss of context

Reply:
synchronization maybe very pool performance. Event your notice is correct, the modification is not good.